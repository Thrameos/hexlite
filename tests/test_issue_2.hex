
d(0). d(1).
a(b) ; n_a(b).
num(X) :- &issue_2_num[a](X), d(X).

