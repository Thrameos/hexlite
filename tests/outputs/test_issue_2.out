{a(b),d(0),d(1),num(1)}
{n_a(b),d(0),d(1),num(0)}
